drivers/i2c/i2c_ll_stm32.c

```diff
@@ -14,7 +14,6 @@
 #include <drivers/i2c.h>
 #include "i2c_ll_stm32.h"
 
-#define LOG_LEVEL CONFIG_I2C_LOG_LEVEL
```

Comment:
This would deserve a dedicated clean up commit.

Reply:
removed

>

Reply:
Still there as I can see

Reply:
lines 
`#define LOG_LEVEL_CONFIG_I2C_LOG_LEVEL`
`#include <logging/log.h>`
`LOG_MODULE_REGISTER(i2c_ll_stm32);`
do not changed